rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Data sharing specifications for the NF community"
+author:
+- name: Robert Allaway
+  url: https://example.com/
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0003-3573-3565
+- name: Jineta Banerjee
+  url: https://example.com/
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0002-1775-3645
+- name: Anh Nguyet Vu
+  url: https://github.com/anngvu
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0003-1488-6730
+date: "`r Sys.Date()`"
+description: |
+  Status: **Draft**
+draft: yes
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(gt)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+
+survey <- read.csv("materials/rfc/rfc_responses.csv")
+```
+
+## Abstract
+
+This document describes data types and formats that should be shared by the Neurofibromatosis (NF) research community for the NF Data Portal. 
+
+## Introduction
+
+The NF Data Portal is a collaborative data-sharing platform for Neurofibromatosis, a rare genetic disease. 
+It has been jointly supported by multiple funding organizations -- the Gilbert Family Foundation (GFF), Children's Tumor Foundation (CTF), Neurofibromatosis Therapeutic Acceleration Program (NTAP), NCI Developmental and Hyperactive Ras Tumor SPORE (DHART SPORE), CDMRP Neurofibromatosis Research Program (CDMRP NFRP), and Neurofibromatosis Research Initiative (NFRI) -- and therefore represents substantial NF research activity.
+
+As the NF community may continue to expand to add new affiliated or independent researchers, and therefore new data, having these standards will help maintain the quality and quantity of our valuable resources. 
```

Comment:
```suggestion
As the NF Data Portal community continues to expand to add newly affiliated as well as independent researchers, and therefore new data, having these standards will help maintain the quality and quantity of our valuable resources. 
```

Reply:
It may be helpful to add a sentence earlier to explain what is being refered to as `these standards`